src/test/scala/ohnosequences/loquat/dataProcessing.scala

```diff
@@ -8,20 +9,27 @@ import better.files._
 
 case object dataProcessing {
 
-  implicit def fileParser[D <: AnyData](implicit d: D): DenotationParser[D,AnyDataLocation, FileDataLocation] = new DenotationParser(d,d.label)( { v: FileDataLocation => Some(v) })
+  implicit def fileParser[D <: AnyData](implicit d: D):
+    DenotationParser[D, AnyDataLocation, FileDataLocation] =
+    new DenotationParser(d, d.label)({ f: FileDataLocation => Some(f) })
 
-  case object instructs extends DataProcessingBundle()(
-    inputData,
-    outputData
+  case class inputData(sample: Sample) extends DataSet(reads1(sample) :×: reads2(sample) :×: |[AnyData])
+  case class outputData(sample: Sample) extends DataSet(stats :×: mergedReads(sample) :×: |[AnyData])
+
+  case class processingBundle(sample: Sample) extends DataProcessingBundle()(
+    inputData(sample), 
+    outputData(sample)
```

Comment:
@eparejatobes please, take a look at this example code. It doesn't encounter an implicit for the parser..